properties/imagepipeline.py

```diff
@@ -93,6 +93,12 @@ class LuxCoreImagepipelineVignetting(PropertyGroup):
     scale: FloatProperty(name="Strength", default=40, min=0, soft_max=60, max=100, precision=1,
                           subtype="PERCENTAGE", description="Strength of the vignette")
 
+class LuxCoreImagepipelineWhiteBalance(PropertyGroup):
+    NAME = "White Balance"
+    enabled: BoolProperty(name=NAME, default=False, description="Enable/disable " + NAME)
+
+    temperature: FloatProperty(name="temperature", default=6500, min=1000, soft_max=10000, max=10000, precision=50,
+                          subtype="PERCENTAGE", description="White point temperature")
```

Comment:
I think this should not be `subtype="PERCENTAGE`?

Also, if `soft_max` and `max` are the same, you can remove `soft_max`.

Reply:
Is it FACTOR the right type?
I don't where to look for the kinds that are supported

Reply:
Just remove the subtype argument, it's a temperature in Kelvin. Or am I not seeing something here?

Reply:
I didn't know I could remove the type. You're correct, it is just a temperature